src/gsCore/gsDofMapper.cpp

```diff
@@ -21,6 +21,8 @@ gsDofMapper::gsDofMapper() :
 m_shift(0), m_numFreeDofs(0), m_numCpldDofs(1), m_curElimId(-1)
 { 
     m_offset.resize(1,0);
+
+    m_isPermuted = false;
```

Comment:
maybe we should set m_isPermuted in the initializer list; that would make it easier to check that it is initialized in all cases.

Reply:
jup, you are right